### ISSUE TITLE
New Options for filtering Date

### ISSUE THREAD

**Issue by LaKanDoR** [piwik/piwik#22330]
New Options for filtering Date

## Summary

    Date filter: last 24 hours
    Date filter: last 30 days
    Date filter: laast 7 Days
    
    Also calculate the last 7 days Also calculate the last 7 days (e.g. from Friday to Friday) and not like today from calendar week complete to calendar week complete, that cannot be compared with the comparison function in principle, so the current week is meant)

including the comparison for the new three date filter options

## Your Environment

* Matomo Version: 5.1 RC1
* PHP Version: 8.3
* Server Operating System: Debian 12.5
* Additionally installed plugins: none

**Comment by Chardonneaur** [comment]
I got someone today asking for this feature.

https://github.com/user-attachments/assets/4a7d27c3-a41e-4031-a90f-8851d99410c7

**Comment by LaKanDoR** [comment]
any progress? 

it would be so cool if we could use it like google analytics, sometimes its hard to follow the statistics.

**Comment by mattab** [comment]
Thanks for the suggestion! It duplicates https://github.com/matomo-org/matomo/issues/18573
